.github/workflows/gammasim-tools-dev.yml

```diff
@@ -46,13 +52,14 @@ jobs:
           tags: |
             type=ref,event=pr,suffix=-gammasim-tools-dev
             type=semver,pattern={{major}}.{{minor}}.{{patch}},suffix=-gammasim-tools-dev
-          images: ${{ env.REGISTRY }}/gammasim-tools-dev/${{ env.IMAGE_NAME }}
+          images: ${{ env.REGISTRY }}/gammasim/${{ env.IMAGE_NAME }}
 
       - name: Build and push Docker image
         uses: docker/build-push-action@v3
         with:
           context: .
-          push: ${{ github.event_name != 'pull_request' }}
+          platforms: linux/amd64,linux/arm64/v8
```

Comment:
Stumbled over the naming  linux/amd64, but it apparently includes also intel machines.

Reply:
Yes, it's just another name for `x86_64`